src/transformers/models/dpt/modeling_flax_dpt.py

```diff
@@ -0,0 +1,1236 @@
+
+    def setup(self):
+        super().__init__()
+        self.layers = FlaxDPTFeatureFusionLayerCollection(self.config, self.dtype)
+
+    def __call__(self, hidden_states):
+        # reversing the hidden_states, we start from the last
+        hidden_states = hidden_states[::-1]
+
+        fused_hidden_states = []
+        # first layer only uses the last hidden_state
+        fused_hidden_state = self.layers(hidden_states[0], residual=None, i=0)
+        fused_hidden_states.append(fused_hidden_state)
+        # looping from the last layer to the second
+        for i, hidden_state in enumerate(hidden_states[1:]):
+            fused_hidden_state = self.layers(fused_hidden_state, residual=hidden_state, i=i + 1)
+            fused_hidden_states.append(fused_hidden_state)
+        return fused_hidden_states
+
+
+class FlaxDPTPreActResidualLayer(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+
+    def setup(self):
+
+        self.use_batch_norm = self.config.use_batch_norm_in_fusion_residual
+        self.activation1 = ACT2FN["relu"]
+        self.convolution1 = nn.Conv(
+            self.config.fusion_hidden_size,
+            kernel_size=(3, 3),
+            strides=(1, 1),
+            padding="SAME",
+            use_bias=not self.use_batch_norm,
+        )
+
+        self.activation2 = ACT2FN["relu"]
+        self.convolution2 = nn.Conv(
+            self.config.fusion_hidden_size,
+            kernel_size=(3, 3),
+            strides=(1, 1),
+            padding="SAME",
+            use_bias=not self.use_batch_norm,
+        )
+
+        if self.use_batch_norm:
+            self.batch_norm1 = nn.BatchNorm(use_running_average=True)
+            self.batch_norm2 = nn.BatchNorm(use_running_average=True)
+
+    def __call__(self, hidden_state):
+        residual = hidden_state
+        hidden_state = self.activation1(hidden_state)
+
+        hidden_state = self.convolution1(hidden_state)
+
+        if self.use_batch_norm:
+            hidden_state = self.batch_norm1(hidden_state)
+
+        hidden_state = self.activation2(hidden_state)
+        hidden_state = self.convolution2(hidden_state)
+
+        if self.use_batch_norm:
+            hidden_state = self.batch_norm2(hidden_state)
+
+        return hidden_state + residual
+
+
+class FlaxDPTFeatureFusionLayerCollection(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+
+    def setup(self):
+        self.layers = [
+            FlaxDPTFeatureFusionLayer(self.config, name=str(i)) for i in range(len(self.config.neck_hidden_sizes))
+        ]
+
+    def __call__(self, hidden_states, residual=None, i=0):
+        return self.layers[i](hidden_states, residual)
+
+
+class FlaxDPTFeatureFusionLayer(nn.Module):
+    config: DPTConfig
+    dtype: jnp.dtype = jnp.float32
+    align_corners: bool = True
+
+    def setup(self):
+        self.projection = nn.Conv(self.config.fusion_hidden_size, kernel_size=(1, 1))  # , bias=True)
```

Comment:
`use_bias` defaults to `True`, but can pass`use_bias=True` as a kwarg for clarity (as done in PT)
```suggestion
        self.projection = nn.Conv(self.config.fusion_hidden_size, kernel_size=(1, 1), use_bias=True)
```